src/login/login.cpp

```diff
@@ -546,7 +546,7 @@ void maint_config_read(const char* key, const char* value)
     {
         maint_config.maint_mode = std::atoi(value);
 
-        if (maint_config.maint_mode > 2 || maint_config.maint_mode < 0)
+        if (maint_config.maint_mode > 2)
```

Comment:
guessing original implementation was copypasta, because this is only ever 0 or 1 and should have been just true/false

Reply:
Got that covered now, good catch